src/Rendering.cpp

```diff
@@ -2,7 +2,7 @@
 #include "Access.h"
 #include "FPNG.h"
 
-#include <SDL.h>
+#include <SDL2/SDL.h>
```

Comment:
This should now be fixed and actually incorrect - pull from main and compile from meson and VSCode